hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraTable.java

```diff
@@ -156,6 +156,12 @@ protected <R> Iterator<R> query(Query query,
         } catch (DriverException e) {
             LOG.debug("Failed to query [{}], detail statement: {}",
                       query, selects, e);
+            // Closing the iterator
+            try {
+                rs.close();
+            } catch (Exception closeException) {
+                throw new BackendException("Error closing Iterator during query [%s]", e, query);
```

Comment:
I think it's better to just log.error closeException here and let throw the DriverException at line 165, because closeException is not the main path.

Reply:
done